spec/rails_helper.rb

```diff
@@ -0,0 +1,66 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
+
+# Load EngineCart's test application.
+EngineCart.load_application!
```

Comment:
I have a `require 'engine_cart'` before the load in my `rails_helper.rb`.  Isn't it necessary here?

Reply:
Hm. It certainly needs to be required _somewhere_ but not sure where it's getting required on my local instance. Let me check.